flight/Modules/System/systemmod.c

```diff
@@ -360,12 +447,18 @@ static void updateWDGstats()
 }
 #endif
 
+#ifdef PIPXTREME
```

Comment:
Weird to have this #ifdef in systemmod.c. Should we even continue to support the PipXtreme? It's not manufactured anymore, right? Or is this more targeting RFMB22? If so, does it make sense to have an #ifdef for a specific hardware chip?